FROM python:3.3


RUN apt-get update
